ExtractTable/common.py

```diff
@@ -24,19 +24,19 @@ def __init__(self, data: dict, fmt: str = DEFAULT, index: bool=False):
         self.data = data
         self.output = self._converter(fmt.lower(), index=index)
 
-    def _converter(self, fmt: str, orient: str = "index", index: bool = False) -> list:
+    def _converter(self, fmt: str, index: bool = False) -> list:
         """
         Actual conversion takes place here using Pandas
         :param fmt: format to be converted into
         :param orient: required orientation of tabualar data in the output
         :param index: row index consideration in the output
         :return: list of tables from converted into the requested output format
         """
-        dfs = [pd.read_json(json.dumps(table["TableJson"]), orient=orient) for table in self.data["Tables"]]
+        dfs = [pd.read_json(json.dumps(table["TableJson"]).T) for table in self.data["Tables"]]
```

Comment:
While the bug identification was correct, the solution is not. The transpose has to be on Dataframe but not on string. Taken care with an update in the revision.